.github/workflows/autoupdate-preview.yaml

```diff
@@ -20,27 +20,28 @@ jobs:
         env:
           API_BASE_URL:  https://raw.githubusercontent.com/mongodb/openapi/refs/heads/dev/openapi/v2
       - name: Verify Changed files
-        uses: tj-actions/verify-changed-files@4a3570896cbdc5dca56a7a259fba88667845e18a
-        id: verify-changed-files
-        with:
-          files: |
-             **/atlas-api.yaml
+        run: |
+          FILES_CHANGED=false
+          if git diff --name-only | grep -qE '^openapi/atlas-api.yaml'; then
+              FILES_CHANGED=true
+          fi
+          echo "FILES_CHANGED=${FILES_CHANGED}" >> "$GITHUB_ENV"
```

Comment:
❤️ Simple, easy, Maintenance free!

Reply:
Ideally we could have logic as script for reuse but in this case this is short enough to have it inline